tests/presentation.test.ts

```diff
@@ -188,81 +185,43 @@ test('presentation with context binding', async (t) => {
         Operation.equals(Operation.property(signedData, 'age'), targetAge),
         Operation.equals(Operation.property(signedData, 'name'), targetName)
       ),
-      data: Operation.property(signedData, 'age'),
+      ouputClaim: Operation.property(signedData, 'age'),
     })
   );
+  const data = { age: Field(18), name: Bytes32.fromString('Alice') };
+  const signedData = Credential.sign(issuerKey, { owner, data });
 
-  await t.test('presentation with zk-app context', async () => {
-    const program = createProgram(spec);
-    const verificationKey = await program.compile();
-    const presentationCircuitVKHash = verificationKey.hash;
-
-    const data = { age: Field(18), name: Bytes32.fromString('Alice') };
-    const signedData = Credential.sign(issuerKey, { owner, data });
-
-    const inputContext = {
-      presentationCircuitVKHash,
-      action: Field(123), // Mock method ID + args hash
-      serverNonce: Field(456),
-    };
-
-    const walletContext = {
-      verifierIdentity: zkAppVerifierIdentity,
-      clientNonce: Field(789),
-    };
```

Comment:
this shows pretty well how the API to specify context changed: we only pass in action (on the server) and verifierIdentity (on the client)